Content-Controls/Add-tick-mark-checkbox/Add-tick-mark-checkbox/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIO;
+
+//Register Syncfusion license
+Syncfusion.Licensing.SyncfusionLicenseProvider.RegisterLicense("Mgo+DSMBMAY9C3t2UlhhQlNHfV5DQmBWfFN0QXNYfVRwdF9GYEwgOX1dQl9nSXZTc0VlWndfcXNSQWc=");
+
+using (WordDocument wordDocument = new WordDocument())
+{
+    // Add a section and paragraph to initialize an empty document structure
```

Comment:
//Add one section and one paragraph to the document.

Reply:
Addressed